chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -113,13 +113,17 @@ dataDir.notFound						= The Zotero data directory could not be found.
 dataDir.previousDir					= Previous directory:
 dataDir.useProfileDir				= Use %S profile directory
 dataDir.selectDir					= Select a Zotero data directory
+dataDir.selectNewDir				= Select a new Zotero data directory
 dataDir.changeDataDirectory 		= Change Data Directory…
+dataDir.pickNewDataDirectory		= Pick New Data Directory…
 dataDir.unsafeLocation.selected.dropbox = Choosing a data directory within Dropbox may corrupt your database.
 dataDir.unsafeLocation.selected.useAnyway = Use this directory anyway?
 dataDir.unsafeLocation.existing.dropbox = Your Zotero data directory is within Dropbox, which may lead to data corruption.
 dataDir.unsafeLocation.existing.chooseDifferent = Would you like to choose a different location now?
 dataDir.selectedDirNonEmpty.title		= Directory Not Empty
 dataDir.selectedDirNonEmpty.text		= The directory you selected is not empty and does not appear to be a Zotero data directory.\n\nCreate Zotero files in this directory anyway?
+dataDir.mustSelectEmpty.title			= Directory Not Empty
+dataDir.mustSelectEmpty.text			= The directory you selected is not empty. You need to select an empty directory to continue.
```

Comment:
"The directory you selected is not empty. You must select an empty directory to continue."